### PR TITLE
Collective Close Operational

### PR DESCRIPTION
**Pull request by shawntabrizi**
This PR creates an explicit `close_operational` transaction that can only be called by collective members in order to close a vote with `Operational` priority. The original `close` extrinsic now has `Normal` priority and can be called by anyone.

This is just a little bit safer against spam attacks from irrational users who may want to waste their money in order to fill a block. In this case, the average user does not have access to generate arbitrary `Operational` transactions.

### CODE REVIEW

**Comment by redzsina** [comment]
We are done with our review for this PR - please refer to issue #7 in the internal tracker.

**Comment by thiolliere** [comment]
> Needs a signed extension

I guess this PR is not ready for review anymore ?

File: frame/collective/src/lib.rs [hunk 1]
```diff
@@ -679,6 +611,61 @@ decl_module! {
 			let proposal_count = Self::do_disapprove_proposal(proposal_hash);
 			Ok(Some(T::WeightInfo::disapprove_proposal(proposal_count)).into())
 		}
+
+		/// Close a vote that is either approved, disapproved, or whose voting period has ended
+		/// with operational transaction priority.
+		///
+		/// May be called only by an active member close the proposal with operational priority.
```
Comment (gavofyork) [inline frame/collective/src/lib.rs:618]:
```suggestion
		/// May be called only by an active member to close the proposal with operational priority.
```

**Review by gavofyork — APPROVED** [review]

**Comment by gnunicorn** [comment]
So, anyone working on this?

**Comment by gnunicorn** [comment]
closing due to a lack of progress.

**Pull request closed without merging**